spec/services/teacher_training_public_api/sync_courses_spec.rb

```diff
@@ -4,6 +4,8 @@
   include TeacherTrainingPublicAPIHelper
 
   describe '.call' do
+    let(:incremental_sync_false) { false }
```

Comment:
Not good. Ideas?
https://github.com/DFE-Digital/apply-for-teacher-training/pull/4712/files#r631177409